lib/compute/operation.js

```diff
@@ -268,6 +274,11 @@ Operation.prototype.startPolling_ = function() {
     }
 
     if (metadata.status !== 'DONE') {
+      if (self.progress !== metadata.progress) {
```

Comment:
I've written an additional test for that - I think it could be cleaner but I'm not sure how to go about doing that.